updates for Swift 3 support

angelodipaolo opened the pull request.

cihancimen commented on ELWebServiceExample/ELWebServiceExample/HTTP/ServiceTask+Brew.swift:
```diff
@@ -9,23 +9,44 @@
 import Foundation
 import ELWebService
 
+//extension ServiceTask {
```
Can we remove commented out code?

cihancimen commented:
Following files have compilation errors due to enumaration case changes.
ELWebServiceTests/ServicePassthroughDelegateTests.swift
ELWebServiceTests/ServiceTaskTests.swift

angelodipaolo commented:
@cihancimen thanks for catching these issues. I removed the commented-out code and updated the unit tests to accommodate the `ServiceTaskResult` case changes.
